Tests failing on 32bit platforms

```
--- FAIL: TestUseStorage (1.03s)
panic: During db.vlog.open err: Error while creating log file in valueLog.open err: while opening file: /tmp/badger-test.badger/000003.vlog err: cannot allocate memory
	while mmapping /tmp/badger-test.badger/000003.vlog with size: 2147483646 [recovered, repanicked]

goroutine 10 [running]:
testing.tRunner.func1.2({0xaaae18, 0xa482080})
	/usr/lib/go-1.26/src/testing/testing.go:1974 +0x26c
testing.tRunner.func1()
	/usr/lib/go-1.26/src/testing/testing.go:1977 +0x3c4
panic({0xaaae18, 0xa482080})
	/usr/lib/go-1.26/src/runtime/panic.go:860 +0x104
github.com/dundee/gdu/v5/pkg/analyze.(*Storage).Open(0x4d5b2c0)
	/build/reproducible-path/gdu-5.34.0/_build/src/github.com/dundee/gdu/v5/pkg/analyze/storage.go:62 +0xf4
github.com/dundee/gdu/v5/pkg/analyze.(*StoredDir).loadFiles(0xa402168)
	/build/reproducible-path/gdu-5.34.0/_build/src/github.com/dundee/gdu/v5/pkg/analyze/stored.go:325 +0xbc
github.com/dundee/gdu/v5/pkg/analyze.(*StoredDir).GetFiles.func1(0xa35a1d0)
	/build/reproducible-path/gdu-5.34.0/_build/src/github.com/dundee/gdu/v5/pkg/analyze/stored.go:302 +0x2c
github.com/dundee/gdu/v5/stdout.(*UI).showDir(0x4d5b100, {0xbb8524, 0xa402168})
	/build/reproducible-path/gdu-5.34.0/_build/src/github.com/dundee/gdu/v5/stdout/stdout.go:253 +0xcc
github.com/dundee/gdu/v5/stdout.(*UI).AnalyzePath(0x4d5b100, {0x4c32600, 0x5c}, {0x0, 0x0})
	/build/reproducible-path/gdu-5.34.0/_build/src/github.com/dundee/gdu/v5/stdout/stdout.go:219 +0x1f8
github.com/dundee/gdu/v5/cmd/gdu/app.(*App).runAction(0x4d5b040, {0xbb808c, 0x4d5b100}, {0x4c32600, 0x5c})
	/build/reproducible-path/gdu-5.34.0/_build/src/github.com/dundee/gdu/v5/cmd/gdu/app/app.go:586 +0x30c
github.com/dundee/gdu/v5/cmd/gdu/app.(*App).Run(0x4d5b040)
	/build/reproducible-path/gdu-5.34.0/_build/src/github.com/dundee/gdu/v5/cmd/gdu/app/app.go:286 +0x6d4
github.com/dundee/gdu/v5/cmd/gdu/app.runApp(0x4d09328, {0x4c02eb8, 0x1, 0x1}, 0x0, {0xbb2a24, 0x1361228})
	/build/reproducible-path/gdu-5.34.0/_build/src/github.com/dundee/gdu/v5/cmd/gdu/app/app_test.go:609 +0x23c
github.com/dundee/gdu/v5/cmd/gdu/app.TestUseStorage(0x4d0aa08)
	/build/reproducible-path/gdu-5.34.0/_build/src/github.com/dundee/gdu/v5/cmd/gdu/app/app_linux_test.go:68 +0xc8
testing.tRunner(0x4d0aa08, 0xba8bb0)
	/usr/lib/go-1.26/src/testing/testing.go:2036 +0x11c
created by testing.(*T).Run in goroutine 1
	/usr/lib/go-1.26/src/testing/testing.go:2101 +0x4f8
FAIL	github.com/dundee/gdu/v5/cmd/gdu/app	1.045s
...
```

See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1131125

--- a/internal/common/const.go
+++ b/internal/common/const.go
@@ -1,0 +1,4 @@
+package common
+
+// Is64Bit returns true if the system is 64-bit
+const Is64Bit = (^uint(0) >> 63) == 1

--- a/pkg/analyze/storage.go
+++ b/pkg/analyze/storage.go
@@ -7,6 +7,7 @@ import (
 	"sync"
 
 	"github.com/dgraph-io/badger/v4"
+	"github.com/dundee/gdu/v5/internal/common"
 	"github.com/dundee/gdu/v5/pkg/fs"
 	"github.com/pkg/errors"
 )
@@ -57,6 +58,13 @@ func (s *Storage) IsOpen() bool {
 func (s *Storage) Open() func() {
 	options := badger.DefaultOptions(s.storagePath)
 	options.Logger = nil
+
+	if !common.Is64Bit {
+		// For 32-bit systems, we need to set ValueLogFileSize to a smaller value to
+		// avoid "cannot allocate memory while mmapping" error
+		options.ValueLogFileSize = (1<<30 - 1) / 2
+	}
+
 	db, err := badger.Open(options)
 	if err != nil {
 		panic(err)

